dist/minimongo-observable/to-observable.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 var observable_cursor_1 = require('./observable-cursor');
+var _ = require('lodash');
+var COLLECTION_EVENTS_DEBOUNCE_TIMEFRAME = 16;
 function toObservable(cursor) {
     var observable = observable_cursor_1.ObservableCursor.create(function (observer) {
```

Comment:
Figured it out. the whole call stack comes from the subscription which should run `next` within the correct zone of execution.